utils/index.j2

```diff
@@ -0,0 +1,18 @@
+<html>
+  <head>
+    <title>Listing</title>
+  </head>
+<body>
+{% if files|length > 1 %}
+  {{ files|length }} entries<br><br>
+{% else %}
+  {{ files|length }} entry<br><br>
```

Comment:
Can we drop the two breaks here? Seems like the `<ul>` adds a nice spacing already!